README.md

```diff
@@ -4,27 +4,110 @@ This document describes how you can collect Corelight Sensor logs by configuring
 
 The architecture diagram shows the following components:
 
-* **Corelight Sensor**: The system running the [Corelight Sensor](https://docs.corelight.com/docs/sensor/sensor/export/syslog.html).
+*  **Corelight Sensor**: The system running the [Corelight Sensor ](https://docs.corelight.com/docs/sensor/sensor/export/syslog.html).
 
-* **The Corelight Sensor exporter**: The [Corelight Sensor exporter](https://docs.corelight.com/docs/sensor/sensor/export/syslog.html) collects log data from the Sensor, and forwards it to the Chronicle forwarder.
+*  **Corelight Sensor exporters**: The [Corelight Sensor exporter](https://docs.corelight.com/docs/sensor/sensor/export/syslog.html) collects log data from the Sensor, and forwards it to the Google Security Operations.
 
-* **Chronicle forwarder**: The Chronicle  forwarder is a lightweight software component, deployed in the customer's network, that supports syslog. The Chronicle  forwarder forwards the logs to Chronicle .
+*  **Google Security Operations**: Google Security Operations retains and analyzes the logs from
+    Corelight Sensor.
 
-*  **Chronicle**: Chronicle retains and analyzes the logs from Corelight Sensor.
+### Configure the Corelight log Exporter for Google SecOps 
 
-An ingestion label identifies the parser which normalizes raw log data to structured UDM format. The information in this document applies to the parser with the `CORELIGHT` ingestion label.
+1. Sign into Corelight Sensor as an adminstrator.
 
-## Before you begin
+2. Select the **Exporters (Dynamic)** tab and select Google SecOps.
+
+  ![Deployment architecture](images/dynamic-exporter-step-1.png)
+
+3. Configure the following input parameters:
+  - **Exporter Name**: the name of the exporter.
+  - **Google SecOps Customer ID**: the customer Id of the Google SecOps.
+  - **Google SecOps Namespace**: the unique namespace associated with Google SecOps for organizing and managing data.
+  - **Google SecOps Labels**: a set of key-value pairs representing the labels.
+  - **Region**: the geographical region where Google SecOps is deployed.
+  - **Credentials**: the authentication details required to securely connect and export data to Google SecOps.
+  - **Proxy URL**: the URL of the proxy server used to route traffic between the exporter and Google SecOps.
+  - **Log Type Filter**: specify whether to include or exclude certain log types.
+  - **Zeek Logs**: select which log types to include or exclude by selecting all applicable options.
+
+  ![Deployment architecture](images/dynamic-exporter-step-2.png)
+
+4. Click **Done**
+
+  ![Deployment architecture](images/dynamic-exporter-step-3.png)
+
+## Ingesting Logs into Google SecOps Using a Forwarder 
+
+  ![Deployment architecture](images/parser-corelight-architecture.png)
+
+The architecture diagram shows the following components:
+
+*  **Corelight Sensor**: The system running the [Corelight Sensor ](https://docs.corelight.com/docs/sensor/sensor/export/syslog.html).
```

Comment:
The link should point to 

https://docs.corelight.com/docs/sensor/sensor/quickstarts/quickstart.html

Reply:
Updated.